Sources/CoreCommands/Options.swift

```diff
@@ -615,25 +615,36 @@ public struct LinkerOptions: ParsableArguments {
 @_spi(SwiftPMInternal)
 public struct TestLibraryOptions: ParsableArguments {
     public init() {}
-
+    
     /// Whether to enable support for XCTest (as explicitly specified by the user.)
     ///
     /// Callers will generally want to use ``enableXCTestSupport`` since it will
     /// have the correct default value if the user didn't specify one.
     @Flag(name: .customLong("xctest"),
           inversion: .prefixedEnableDisable,
           help: "Enable support for XCTest")
-    public var explicitlyEnableXCTestSupport: Bool?
+    public var enableXCTestSupport: Bool = false
 
+    private var didSetEnableXCTestSupport = false
+
+    public var explicitlyEnableXCTestSupport: Bool? {
+        didSetEnableXCTestSupport ? enableXCTestSupport : nil
+    }
+    
     /// Whether to enable support for Swift Testing (as explicitly specified by the user.)
     ///
     /// Callers will generally want to use ``enableSwiftTestingLibrarySupport`` since it will
     /// have the correct default value if the user didn't specify one.
     @Flag(name: .customLong("swift-testing"),
           inversion: .prefixedEnableDisable,
           help: "Enable support for Swift Testing")
-    public var explicitlyEnableSwiftTestingLibrarySupport: Bool?
+    public var enableSwiftTestingLibrarySupport: Bool = true
 
+    private var didSetEnableSwiftTestingLibrarySupport = false
+    
+    public var explicitlyEnableSwiftTestingLibrarySupport: Bool? {
+        didSetEnableSwiftTestingLibrarySupport ? enableSwiftTestingLibrarySupport : nil
+    }
```

Comment:
Properties should be separated with a newline

```suggestion
    }

```